### ISSUE TITLE
Transitive dependencies for pods

### ISSUE THREAD

**Issue by leo-neon** [PDFTron/pdftron-flutter#280]
DESCRIPTION:
For the PDFTron integration on iOS we need to add some pods to the Podfile.

```
pod 'PDFTron', podspec: 'https://pdftron.com/downloads/ios/flutter/pdftron/latest.podspec'
pod 'PDFTronTools', podspec: 'https://pdftron.com/downloads/ios/flutter/pdftron-tools/latest.podspec'

```

I get the following error for a **pod install**: 
[!] The 'Pods-Runner' target has transitive dependencies that include statically linked binaries: (...../ios/Pods/PDFTron/PDFNet.xcframework)

I know from other sources, that you need to remove the line 
`use_frameworks!`
in order to make the pod installation work in that case, but other packages in our project depend on that line, so we cannot remove it.

REPRODUCTION:
- Add the mentioned pods to Podfile and also `use_frameworks!`
- Run `pod install` in iOS folder

EXPECTED:
The pods integration should also work with use_frameworks.

PLATFORM/DEVICE
 - Platform: iOS
 - Flutter: 3.0.5

**Comment by Tyler-Gordon** [comment]
Hello leo-neon,
I found a very similar issue with a fix here: https://stackoverflow.com/a/52381474

Let us know if this works for you

**Comment by leo-neon** [comment]
Hi Tyler,
thanks for giving me this idea again. I actually tried it before and added `s.static_framework = true` to the pdftron_flutter.podspec. After a pod install, the Xcode build succeeded but the app crashed instantly after launch. 

What I did this time is to deintegrate all pods before, so that everything gets reinstalled. Totally forgot about that and now it works👍

Thank you

**Comment by leo-neon** [comment]
Alright, maybe this helps some people with the same problem. After my fix, I also tried to run the app in release mode. Didn't work and I got no logs in the Flutter Debugger. Running it in Xcode again, it showed me the error logs which I mentioned earlier. Some "library is not loaded" and the logs were concerning the PDFNet.framework.

I'm not quite sure, whether the following did the trick, but I opened Xcode, head over to my target's General tab. Under Frameworks I explicitly embedded the PDFNet.xcframework. Now I can run the app in debug and release mode.
